docs/ml-features.md

```diff
@@ -53,7 +53,10 @@ collisions, where different raw features may become the same term after hashing.
 chance of collision, we can increase the target feature dimension, i.e. the number of buckets 
 of the hash table. Since a simple modulo is used to transform the hash function to a column index, 
```

Comment:
Should we also mention that we use ``` Austin Appleby's MurmurHash 3 algorithm``` to calculate the hash code value? Because how to set the feature dimension is related with the hash algorithm.